engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterSceneDelegate.m

```diff
@@ -29,4 +30,54 @@ - (void)scene:(UIScene*)scene
   }
 }
 
+- (void)windowScene:(UIWindowScene*)windowScene
+    performActionForShortcutItem:(UIApplicationShortcutItem*)shortcutItem
+               completionHandler:(void (^)(BOOL succeeded))completionHandler {
+  id appDelegate = FlutterSharedApplication.application.delegate;
+  if ([appDelegate respondsToSelector:@selector(lifeCycleDelegate)]) {
+    FlutterPluginAppLifeCycleDelegate* lifeCycleDelegate = [appDelegate lifeCycleDelegate];
+    [lifeCycleDelegate application:FlutterSharedApplication.application
+        performActionForShortcutItem:shortcutItem
+                   completionHandler:completionHandler];
+  }
+}
+
+static NSDictionary<UIApplicationOpenURLOptionsKey, id>* ConvertOptions(
```

Comment:
nit: `FLTConvertSceneOpenURLOptions`

Reply:
FYI that is a static function which means it only has file scope, the symbol isn't exported so there is no risk that it will collide with other functions.  Those sorts of naming conventions are important for functions that have external linkage.

Reply:
> the symbol isn't exported so there is no risk that it will collide with other functions

Say you have 

```
static void foo() {
  // print foo
}
```

Then one of the included header has: 
```
void foo() {
  // print bar
}
```
This won't compile right? (haven't tried)

Reply:
Also, something named `foo` in a header would be an exported symbol which should follow a safe naming scheme.